### ISSUE TITLE
Make it more `infinite`

### ISSUE THREAD

**Issue by dora-gt** [PeachScript/vue-infinite-loading#302]
Currently it is not actually `infinite` because all the components are rendered.
Please check [`v-virtual-scroll`](https://github.com/vuetifyjs/vuetify/blob/501005bb8f491a82a06e0c0291d83a0c45b55802/packages/vuetify/src/components/VVirtualScroll/VVirtualScroll.ts) out and integrate it into `vue-infinite-loading`.

**Comment by trandaison** [comment]
I agree! This would be perfect infinite loading, because other virtual infinite loading is not working with the body scroller.
